### ISSUE TITLE
Update Centreon from 2.7.11 to lastest with yum

### ISSUE THREAD

**Issue by atarpin** [centreon/centreon#5295]
Hi,

I follow the step from here : https://documentation.centreon.com/docs/centreon/en/2.8.x/upgrade/from_packages.html

Every thing was ok until this part : Conclude update via Centreon web interface

The thing is that the wizard show nothing in the "4 - installation" part, and the next button was available to click...

I wait for something like 10min but nothing happen, so I click on the "next" button and the final part was "Congratulations, use have successfully upgraded to centreon version **2.7.11**"

And the worst part : when I come back on the web interface every thing was lost... 
Except the logs (in Administration/Logs)

I didn't find how to relaunch the wizard or recover every monitoring I have.

So in conclusion : I lost everything, and the Centreon is still in 2.7.11....

Any solution ?

Do you know if there is a place where my old config could be ? 

Thank you

Arno

EDIT : I put the wrong link

**Comment by atarpin** [comment]
Oh no my bad, my config is still here but I cannot export with the poller I got this error 

`Generating files... NOK (Exception received : SQLSTATE[42S22]: Column not found: 1054 Unknown column 'host_recovery_notification_delay' in 'field list' [file: /usr/share/centreon/www/class/config-generate/host.class.php] [line: 218] )`

I'm afraid my entire Centreon is broken...

**Comment by loiclau** [comment]
hi,
an upgrade script missed, it was added since.
you can upgrade to the latest version.
